src/util/clickTracking.js

```diff
@@ -0,0 +1,154 @@
+const DATA_ATTR = 'clicktrack';
+
+function cleanTrackingUrl() {
+	// TODO: investigate and refactor to support this feature
+
+	/*
+	// remove tracking information from the url to prevent bad link copying
```

Comment:
Just curious - does this happen in meetup classic at the moment, or how is it handled there?